README.md

```diff
@@ -6,58 +6,62 @@
 * **[Pipeline Agnostic](docs/examples.md):** By bundling all logic in bitops, you can have the same experience regardless of which pipeline service runs your CI. You can even run bitops locally!
+* **[Customizable](docs/plugins.md):** Configure how what tools you want installed in your BitOps image. Only take what you need, leave the bloat behind. 
 
 ## How it works
 
-BitOps is a boiler plate docker image for DevOps work. An operations repository is mounted to a bitops image's `/opt/bitops_deployment` directory. BitOps will
+BitOps is a tool orchestrator packaged in a docker image for DevOps work. An operations repository is mounted to a bitops image's `/opt/bitops_deployment` directory. BitOps will parse through the operations repo and;
 
-* Auto-detect any configuration belonging to one of its [supported tools](#supported-tools)
+* Auto-detect bitops configuration files within tool directories
 * Loop through each tool and
+  * Read in `yaml` configuration
   * Run any pre-execute hooks
-  * Read in `yml` configuration
   * Execute the tool
   * Run any post-execute hooks
 
-## Run BitOps
+## Quick Start
 BitOps is packaged as a docker image and is available on [dockerhub](https://hub.docker.com/r/bitovi/bitops).
 ```
-docker pull bitovi/bitops
+docker pull bitovi/bitops:2.0.0-latest
 cd $YOUR_OPERATIONS_REPO
-docker run bitovi/bitops -v .:/opt/bitops_deployment
+docker run bitovi/bitops:2.0.0-latest -v .:/opt/bitops_deployment
 ```
 
-## Configure BitOps
+Need a hand? We got you! Check out our [Example Operation Repos](https://github.com/bitovi/operations-test)
 
+## Configure BitOps
 BitOps is configured in 3 steps:
 
-1. Select your environment
-2. Configure aceess to your cloud provider
+1. Select an environment to process
+2. Configure access to your cloud provider
 3. Configure how you want your deployment tools to execute
 
-[Docs](docs/configuration-base.md)
+Ready to dive deeper? Check out our [Docs](docs/configuration-base.md)
 
-## Supported Tools
-* [Provision infrastructure with CloudFormation](docs/tool-configuration/configuration-cloudformation.md)
-* [Provision infrastructure with Terraform](docs/tool-configuration/configuration-terraform.md)
-* [Configure infrastructure with Ansible](docs/tool-configuration/configuration-ansible.md)
-* [Deploy to kubernetes with Helm](docs/tool-configuration/configuration-helm.md)
+Still not enough? Why not try building and running a [local version of bitops](docs/development-local.md)
 
-## Supported Cloud Providers
+## Supported Plugins
+* [Provision infrastructure with CloudFormation](https://github.com/bitops-plugins/cloudformation/blob/main/README.md)
+* [Provision infrastructure with Terraform](https://github.com/bitops-plugins/terraform/blob/main/README.md)
+* [Configure infrastructure with Ansible](https://github.com/bitops-plugins/ansible/blob/main/README.md)
+* [Deploy to kubernetes with Helm](https://github.com/bitops-plugins/helm/blob/main/README.md)
```

Comment:
Maybe just have the link text be the plugin name rather than providing a description prefix like "Provision infra with .." ?

Ansible, for example, could be used to provision infra, TF could be used to deploy to k8s with helm